src/pages/OrdersPage/Orders.tsx

```diff
@@ -0,0 +1,54 @@
+import AddIcon from '@mui/icons-material/Add';
+import RemoveIcon from '@mui/icons-material/Remove';
+import { Button, ButtonGroup } from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { addProduct, removeProduct } from '../../redux/reducers/productSlice';
+import { RootState } from '../../redux/store';
+import * as Styled from './Orders.styles';
+
+const Orders = () => {
+    const { products: selectedProducts } = useSelector(
+        (state: RootState) => state.products
+    );
+    const dispatch = useDispatch();
+
+    const handleSubmit = () => {
+        /**TODO:IMPLEMENT SUBMITTING */
```

Comment:
To będzie zaimplementowane tu jak coś: https://linear.app/pis22z/issue/PIS-28/fe